electrumsv/wallet.py

```diff
@@ -5023,6 +5024,11 @@ async def _consume_tip_filter_matches_async(self, state: ServerConnectionState)
         """
         Process tip filter messages received from a server.
 
+        This notification pathway has only one intended outcome:
```

Comment:
Delete this comment, it is wrong. We do ask for merkle proofs here in the correct context.

Reply:
With the current code, that could only ever happen if the transaction never hits the mempool and goes straight into a block... But sure - can delete this comment.